apps/machine/zynqmp/platform_info.c

```diff
@@ -94,6 +94,10 @@ struct remoteproc_priv rproc_priv = {
 
 static struct remoteproc rproc_inst;
 
+/* External functions */
+extern int init_system(void);
```

Comment:
Are these still used?  Should we rename to platform_init/platform_cleanup

Reply:
> Are these still used? Should we rename to platform_init/platform_cleanup

This is internal the platform implementation, the reason we do this is because in our case, we have common in implementation for the same platform across different systems, but each systems needs different systems specific initialization.